apps/omg/lib/omg/state/core.ex

```diff
@@ -161,16 +183,25 @@ defmodule OMG.State.Core do
   """
   @spec exec(state :: t(), tx :: Transaction.Recovered.t(), fees :: Fees.optional_fee_t()) ::
           {:ok, {Transaction.tx_hash(), pos_integer, non_neg_integer}, t()}
-          | {{:error, Validator.exec_error()}, t()}
+          | {{:error, Validator.process_error()}, t()}
```

Comment:
naming nitpick for this and `can_process` function. I fear of overloading the process word - is this an erlang process, exit process, process process...

The renamings I'd propose:
`Validator.can_apply_spend` -> `Validator.can_apply_tx`
`Validator.exec_error` -> `Validator.can_apply_error`
`State.Core.apply_spend` -> `State.Core.apply_tx`

Reply:
Regarging `can_process` full fn signature use just once in `State.Core` looks quite OK
```Validator.can_process(state, tx, fees)```

I will rename to `can_process_tx`